### PR TITLE
IR move LUTs to top level

### PR DESCRIPTION
**Pull request by sseifeln**
Look-up tables moved to the top level function to simplify IR-VMR chain.

### CODE REVIEW

File: TrackletAlgorithm/InputRouter.h [hunk 1]
```diff
@@ -316,6 +337,4 @@ void InputRouter( const BXType bx
 }
 
 
-#endif
-
-
+#endif
```
Comment (aehart) [inline TrackletAlgorithm/InputRouter.h:340]: Could you please add a newline to the end of the file?

File: TrackletAlgorithm/InputRouterTop.cc [hunk 2]
```diff
@@ -4,33 +4,17 @@
...
-      , kPhiCorrtable_L2
-      , kPhiCorrtable_L3
       , hInputStubs
       , bx_o
       , hOutputStubs);
-
-  //delete hOutputStubs;
-  //hOutputStubs = cLclCpy;
-}
+}
```
Comment (aehart) [inline TrackletAlgorithm/InputRouterTop.cc:20]: Could you please add a newline to the end of the file?

File: project/script_IR.tcl [hunk 3]
```diff
@@ -29,8 +29,8 @@ create_clock -period 360MHz -name fast_clock
 set nProc [exec nproc]
 csim_design -mflags "-j$nProc" -argv "--link,6 --dtcSplit,0 --tkNonant,4"
 csynth_design 
-# possible options -trace_level all -rtl verilog -verbose 
+# # possible options -trace_level all -rtl verilog -verbose 
 cosim_design  -argv "--link,6 --dtcSplit,0 --tkNonant,4"
-# possible options  -flow syn, -flow impl
+# # possible options  -flow syn, -flow impl
```
Comment (aehart) [inline project/script_IR.tcl:34]: Could you please undo these changes, since they don't affect anything?

Comment (sseifeln) [reply project/script_IR.tcl:34]: Sorry .. last commit was supposed to un-do that instead I commented them out a second time!

File: TrackletAlgorithm/InputRouterTop.cc (hunk 2 above)
Comment (sseifeln) [reply TrackletAlgorithm/InputRouterTop.cc:20]: applied to latest commit.

File: TrackletAlgorithm/InputRouter.h (hunk 1 above)
Comment (sseifeln) [reply TrackletAlgorithm/InputRouter.h:340]: applied to latest commit.

Comment (sseifeln) [reply TrackletAlgorithm/InputRouter.h]: should be fixed in 1055d5ee68c7c31bf35a9d2d940b5a9668b82483

File: TrackletAlgorithm/InputRouterTop.cc (hunk 2 above)
Comment (sseifeln) [reply TrackletAlgorithm/InputRouterTop.cc]: should be fixed in 1055d5ee68c7c31bf35a9d2d940b5a9668b82483

File: project/script_IR.tcl (hunk 3 above)
Comment (sseifeln) [reply project/script_IR.tcl:34]: Fixed in latest commit : 1055d5ee68c7c31bf35a9d2d940b5a9668b82483

Comment (aehart) [reply project/script_IR.tcl]: That did it!

**Comment by aehart** [comment]
This looks good to me, now that my inane comments have been addressed. I'm ready to approve, unless @meisonlikesicecream has any comments to add.

**Comment by meisonlikesicecream** [comment]
> This looks good to me, now that my inane comments have been addressed. I'm ready to approve, unless @meisonlikesicecream has any comments to add.

Nope, I think it's fine!

**Review by meisonlikesicecream — APPROVED** [review]
